ttl/ttlworker/task_manager.go

```diff
@@ -474,6 +498,20 @@ func (m *taskManager) checkInvalidTask(se session.Session) {
 	m.runningTasks = ownRunningTask
 }
 
+func (m *taskManager) reportMetrics() {
+	var runningTasks, finishedTask float64
+	for _, task := range m.runningTasks {
+		switch task.Status {
+		case cache.TaskStatusRunning:
+			runningTasks++
+		case cache.TaskStatusFinished:
+			finishedTask++
+		}
+	}
+	metrics.RunningTaskCnt.Set(runningTasks)
+	metrics.FinishedTaskCnt.Set(finishedTask)
```

Comment:
Is it true that the finished task will be always true because it will be removed immediately once finished

Reply:
Yes. I rewrote this function to set the count with `len(m.runningTasks)` directly.